tasks/preflight.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Ensure postgresql_version is not empty
+  assert:
+    that: postgresql_version | default(false)
```

Comment:
This is mind boggling. I am not an expert on Python, but other people who I consider to be experts, have deemed this to be okay:
https://github.com/ansible/ansible-lint/issues/457#issuecomment-573137057

Reply:
That is pretty damn bazaar for sure. But works for me!